### PR TITLE
Makes The Readme.md better

### PR DESCRIPTION
**Pull request by rajarshimaitra**
This is a proposed change to Readme file for better navigation of new landers. Discussed in [Issue #11](https://github.com/bitcoindevkit/bdk/issues/11)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +8,15 @@ bitcoin network.
...
 1. [Install rustup](https://www.rust-lang.org/learn/get-started)
 
-1. Clone [bitcoindevkit/murmel](https://github.com/bitcoindevkit/murmel) to directory `../murmel` relative to the 
-`./bdk` project repo.
+2. Clone the directory using 
+   ```
+   git clone https://github.com/bitcoindevkit/bdk.git
+   ```
+
+1. Clone bitcoindevkit/murmel using
```
Comment (notmandatory) [inline README.md:16]: you can remove this step now as I've changed the cargo dependency to use the rust-bitcoin/murmel repo directly

File: README.md [hunk 2]
```diff
@@ -90,11 +95,20 @@ bitcoin network.
 1. Build Rust library files for all target platform OS architectures:
     
    ```
+   cd bdk
    ./build-lib.sh
    ```
+1. Check if all the tests are passing
+    
+   ```
+   cargo test --features android
```
Comment (notmandatory) [inline README.md:104]: I think it's enough here to just do:
```
cargo test
```
we don't currently have a way to test jni from rust, and the android feature requires the android logging lib which is only available on an android device

File: README.md [hunk 3]
```diff
@@ -90,11 +95,20 @@ bitcoin network.
 1. Build Rust library files for all target platform OS architectures:
     
    ```
+   cd bdk
    ./build-lib.sh
    ```
+1. Check if all the tests are passing
```
Comment (notmandatory) [inline README.md:101]: I think it's enough here to just do:
```
cargo test
```
we don't currently have a way to test jni from rust, and the android feature requires the android logging lib which is only available on an android device

File: README.md [hunk 4]
```diff
@@ -90,11 +95,20 @@ bitcoin network.
...
    
 ## REGTEST Testing
 
 1. Clone [bitcoin-regtest-box project](https://github.com/bitcoindevkit/bitcoin-regtest-box) and follow
    [README.md](https://github.com/bitcoindevkit/bitcoin-regtest-box/blob/master/README.md) instructions to start 
-   localhost REGTEST bitcoind nodes.
\ No newline at end of file
+   localhost REGTEST bitcoind nodes.
+   
+   Note: regtest-box only checks against bitcoin-core for now. The same will be updated later for bdk.
```
Comment (notmandatory) [inline README.md]: I'm not sure what this note mean, can you say it a different way? or let's chat on slack about it.

File: README.md (hunk 1 above)
Comment (rajarshimaitra) [reply README.md:16]: Done

File: README.md (hunk 3 above)
Comment (rajarshimaitra) [reply README.md:101]: Done

**Comment by rajarshimaitra** [comment]
@notmandatory I have fixed the conflict and rebased. Updated the regtest-box Note. Hopefully, now it makes more sense.

**Comment by notmandatory** [comment]
I decided to replace the regtest-box testing tool with one called nigiri suggested in PR #6 . If you have a chance check it out, it does much more than regtest-box did.  

And looks like this one is out of sync on a couple other spots in the readme. I'll make some comments on which changes should be removed from this PR after you rebase it.

**Comment by rajarshimaitra** [comment]
Rebased and merged the nigiri changes. I will try to check the suit out.

File: README.md (hunk 1 above)
Comment (notmandatory) [reply README.md:16]: @rajarshimaitra maybe you forgot to push your update to your rajarshimaitra:patch-1 branch. Lines 16-23 still need to be removed from this PR.

File: README.md [hunk 5]
```diff
@@ -8,6 +8,19 @@ bitcoin network. The library can be used in an android mobile app by including t
 
 1. [Install rustup](https://www.rust-lang.org/learn/get-started)
 
+2. Clone the directory using
```
Comment (notmandatory) [inline README.md]: If you number this 1. then the markdown tool will auto-number the item. It's better that way so we don't need to keep track of what number we're on when adding new items.

Comment (rajarshimaitra) [reply README.md]: Done.

**Comment by notmandatory** [comment]
closing PR due to move to `magical-bitcoin-wallet` based code for `bdk`

**Pull request closed without merging**